code/jumpnode/jumpnode.cpp

```diff
@@ -177,23 +177,23 @@ object *CJumpNode::GetSCPObject()
 /**
  * @return Color of jump node when rendered
  */
-color CJumpNode::GetColor()
+const color *CJumpNode::GetColor() const
```

Comment:
Why is this a pointer now?  Why not a const reference?

Reply:
Mostly just to be consistent with `GetSCPObject()` and `GetPosition()`.

Reply:
I guess I'm just not convinced that this is a good reason to use a pointer over a reference here.